src/app/Components/InputField.svelte

```diff
@@ -0,0 +1,14 @@
+<script lang="ts">
+  export let label: string;
+  export let id: string;
+  export let value: any;
+  export let subtext: string;
+</script>
+
+<div class="input">
+  <h3>{label}</h3>
```

Comment:
**Suggestion**: if this is the header of the page this h3 is not a good practice for accessibility